elements/bulbs-carousel/bulbs-carousel.test.js

```diff
@@ -98,7 +98,7 @@ describe('<bulbs-carousel>', () => {
     });
 
     context('called twice', () => {
-      it.only('does not create a second track', () => {
+      it('does not create a second track', () => {
```

Comment:
This was causing the test suite to run only this test and skip all others.  So, yeah...